doc/api/test.md

```diff
@@ -2240,6 +2240,47 @@ test('mocks a builtin module in both module systems', async (t) => {
 });
 ```
 
+### `mock.value(object, valueName, value)`
```

Comment:
Is there any reason to make `value` required? In some of the other mocking functions, we make this optional so that we can spy on the existing value instead of needing to provide a new one.